java/AxivityReader.java

```diff
@@ -16,16 +20,25 @@
  */
 public class AxivityReader extends DeviceReader {
 
+    private static ZonedDateTime sessionStart = null;
```

Comment:
@chanshing - can you move this block of 4 variable declarations to the abstract DeviceReader.java class?

i.e. I'm pretty sure we would like this to be available to all of AxivityReader.java, ActigraphReader.java, etc.